Sources/StreamChat/Controllers/PollController/PollVoteListController.swift

```diff
@@ -45,7 +45,7 @@ public class PollVoteListController: DataController, DelegateCallable, DataStore
         return pollVotesObserver.items
     }
     
-    public private(set) var hasLoadedAllVotes: Bool = false
+    public var hasLoadedAllVotes: Bool { votesCursor == nil }
```

Comment:
I think we should not do this.  I also had this bug on the Thread v2. Initially, the cursor will always be `nil`. So, until we fetch the first page, this will be misleading and say that we loaded all votes .
```suggestion
    public var hasLoadedAllVotes: Bool = false
```